app/models/annotation/AnnotationIdentifier.scala

```diff
@@ -21,4 +21,9 @@ object AnnotationIdentifier extends FoxImplicits {
       typ <- AnnotationType.fromString(typ) ?~> ("Invalid AnnotationType: " + typ)
     } yield AnnotationIdentifier(typ, identifier)
 
+  def fromAnnotation(annotation: Annotation)(implicit ec: ExecutionContext): Fox[AnnotationIdentifier] =
+    for {
+      typ <- AnnotationType.fromString(annotation.typ.toString)
```

Comment:
@fm3 I wonder if this could be shortened somehow.

Reply:
Yes, I’d say thid method could just be `def fromAnnotation(annotation: Annotation): AnnotationIdentifier = AnnotationIdentifier(annotation.typ, annotation._id)`

Reply:
Removed.